cmd/builder/internal/command.go

```diff
@@ -102,6 +107,11 @@ func initConfig() error {
 	}
 	cfg.Logger.Info("Using config file", zap.String("path", vp.ConfigFileUsed()))
 
+	// if the include_core flag is set from the config, create a warning message that this is deprecated and will be removed in a subsequent release
+	if vp.IsSet("dist.include_core") {
+		cfg.Logger.Warn("IncludeCore is deprecated, starting from v1.41.0, core components are not going to be implicitly included ")
```

Comment:
On a second thought, I think we need to have this message even if people are not using this flag, as not having the flag implies that core is included, and we are changing the behavior. Perhaps we can have something like the following as part of `config#Validate`.

```go
	// create a warning message that include_core is deprecated and will be removed in a subsequent release
	if c.Distribution.IncludeCore {
		c.Logger.Warn("IncludeCore is deprecated. Starting from v0.41.0, you need to include all components explicitly.")
	} else {
		c.Logger.Info("IncludeCore is deprecated, starting from v0.41.0. The new behavior won't affect your distribution, just remove the option from the manifest.")
	}
```

Reply:
Yes, thank you for pointing this out. I have included this as part of `config#Validate`.